Firebase/android/app/src/main/res/layout/merge_chat_view.xml

```diff
@@ -12,42 +12,33 @@
     <android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:tools="http://schemas.android.com/tools"
       android:id="@+id/messagesRecyclerView"
+      style="@style/Chat.RecyclerView"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:background="@color/background_grey"
-      android:clipToPadding="false"
-      android:paddingBottom="72dp"
       tools:context="com.novoda.bonfire.chat.ChatActivity" />
 
     <LinearLayout
+      style="@style/Chat.NewMessage"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_gravity="bottom"
-      android:layout_marginBottom="@dimen/chat_entry_margin"
-      android:layout_marginLeft="@dimen/chat_entry_margin"
-      android:layout_marginRight="@dimen/chat_entry_margin"
-      android:background="@drawable/oval_shape_white"
-      android:elevation="4dp"
-      android:gravity="center_vertical"
-      android:orientation="horizontal"
```

Comment:
I think orientation would stay here in the layout. ["You shouldn’t have to open styles.xml to diagnose an issue with the layout XML"](You shouldn’t have to open styles.xml to diagnose an issue with the layout XML)